content-repo/extra-docs/releases/20.12.1.md

```diff
@@ -0,0 +1,1284 @@
+If compromised hosts are found then:
+- Notify security team to review and trigger remediation response actions - Fire off sub-playbooks to isolate/quarantine infected hosts/endpoints and await further actions from the security team.
+Note: This is a beta pack, which lets you implement and test pre-release software. Since the pack is beta, it might contain bugs. Updates to the pack during the beta phase might include non-backward compatible features. We appreciate your feedback on the quality and usability of the pack to help us identify issues, fix them, and continually improve.
+
+
+---
+
+### New: Sophos Central Pack v1.0.0
+#### Classifiers
+##### Sophos Central - Classifier
+Classifies Sophos Central incidents
+##### Sophos Central - Incoming Mapper
+Maps incoming alerts from Sophos Central
+#### Incident Types
+- **Sophos Central Incident**
+
+#### Integrations
+##### Sophos Central
+The unified console for managing Sophos products.
+
+
+
+---
+
+### New: Tripwire Pack v1.0.0
+#### Classifiers
+##### Tripwire - Classifier
+Classifies Tripwire incidents.
+##### Tripwire
+
+##### Tripwire - Incoming Mapper
+Maps incoming Tripwire incident fields.
+#### Incident Types
+- **Tripwire File Change**
+
+#### Integrations
+##### Tripwire
+Tripwire is a file integrity management (FIM), FIM monitors files and folders on systems and is triggered when they have changed.
+#### Layouts
+- **Tripwire File Change - Summary**
+
+
+#### Playbooks
+##### Tripwire Incident IP Enrichment
+Enrich the IP of a node.
+
+
+---
+
+### New: Windows Remote Management Pack v1.0.0
+#### Integrations
+##### Windows Remote Management (Beta)
+Uses the Python pywinrm library and commands to execute either a process or using Powershell scripts.
+
+
+---
+
+### New: X509Certificate Pack v1.0.0
+#### Incident Fields
+- **Certificate Names**
+
+- **Certificate Signature**
+
+- **Certificate Validation Checks**
+
+- **Extension**
+
+- **Issuer DN**
+Issuer Distinguished Name
+- **PEM**
+Certificate in PEM format.
+- **Public Key**
+
+- **SPKI SHA256**
+SHA256 fingerprint of Subject Public Key Info
+- **Serial Number**
+
+- **Subject Alternative Names**
+
+- **Subject DN**
+Subject Distinguished Name
+- **Validity Not After**
+End of certificate validity period
+- **Validity Not Before**
+Starting of certificate validity period
+#### Indicator Types
+- **Packs/X509Certificate/IndicatorTypes/reputation-Certificate.json**
+
+#### Layouts
+- ** - None**
+
+#### Scripts
+##### CertificateExtract
+Extract fields from a certificate file and return the standard context.
+##### CertificateReputation
+Enrich and calculate the reputation of a certificate indicator.
+
+
+---
+
+### ARIAPacketIntelligence Pack v2.0.0 (Partner Supported)
+#### Integrations
+##### ARIA Packet Intelligence
+- **Breaking changes:** Update Integration to match the latest Packet Intelligence REST API. Removed **label_sia_region**, **label_sia_group** and **label_sia_name** arguments.
```

Comment:
there are a few formats for `Breaking changes` in these rn, please finalize the format with @ShirleyDenkberg and make sure we are consistent